cypress/support/commands.ts

```diff
@@ -54,7 +51,12 @@ Cypress.Commands.add('mount', (component, options = {}) => {
        * (This is why this is disabled for now)
        *
        */
-      // app.use(options.router)
+      if (options.router) {
+        app.use(options.router)
+      } else {
+        options.router = defaultRouter
+        options.global.components.RouterLink = RouterLink
+      }
```

Comment:
question: just verifying this was copied from the private repo?

Reply:
Kind of, but in the private repo it is
```ts
if (options.router) {
  app.use(options.router)
} else {
  app.use(defaultRouter) // different here
  options.global.components.RouterLink = RouterLink
}
```

If I use `app.use(defaultRouter)` here, [this test](https://github.com/Kong/public-ui-components/blob/3ab908c4b5250e67dcbe36a834d394b60ba3be6e/packages/portal/spec-renderer/src/components/SpecRenderer.cy.ts#L85) will fail. Considering it was `options.router = defaultRouter` [in the first place](https://github.com/Kong/public-ui-components/blob/3ab908c4b5250e67dcbe36a834d394b60ba3be6e/cypress/support/commands.ts#L38), I kept it as is.